src/components/info-window.tsx

```diff
@@ -98,7 +98,7 @@ export const InfoWindow = (props: PropsWithChildren<InfoWindowProps>) => {
       openOptions.anchor = anchor;
     }
 
-    if (shouldFocus) {
+    if (shouldFocus === true || shouldFocus === false) {
```

Comment:
in case its used in typescript that probably holds true, 
what I committed is just in case you use it with vanilla javascript, then it makes sure its a boolean, aka is either true or flase

Reply:
I just think we shouldn't invent our own rules here, if the google maps API will accept `0` and `1` (it probably does), so should we.

Reply:
fair enough ill make the change

Reply:
changed now :)